### PR TITLE
Resolve conflict on Gitlocalize and format markdown

### PR DESCRIPTION
**Pull request by sotayamashita**
:muscle: w/ GitLocalize

### CODE REVIEW

File: src/content/ja/fundamentals/performance/poor-connectivity/index.md [hunk 1]
```diff
@@ -69,15 +69,15 @@ Facebook の [Augmented Traffic Control](http://facebook.github.io/augmented-tra
 
 ![Facebook の Augmented Traffic Control](images/augmented-traffic-control.png)
 
-Facebook は [2G Tuesdays](https://code.facebook.com/posts/1556407321275493/building-for-emerging-markets-the-story-behind-2g-tuesdays/) を設けて、2G を利用するユーザーが自社製品を使用する方法を理解できるようにもしています。毎週火曜日に、従業員にポップアップが表示され、2G 接続をシミュレートするためのオプションが示されます。
+> Facebook は [2G Tuesdays](https://code.facebook.com/posts/1556407321275493/building-for-emerging-markets-the-story-behind-2g-tuesdays/) を設けて、2G を利用するユーザーが自社製品を使用する方法を理解できるようにもしています。毎週火曜日に、従業員にポップアップが表示され、2G 接続をシミュレートするためのオプションが示されます。
 
 [Charles](https://www.charlesproxy.com/){: .external } HTTP/HTTPS プロキシを使用すると、[帯域幅とレイテンシを調整](http://www.charlesproxy.com/documentation/proxying/throttling/)できます。Charles は商用ソフトウェアですが、無料トライアルを利用できます。
 
-![Charles プロキシの帯域幅とレイテンシの設定](images/charles.png)
+![Charles proxy bandwidth and latency settings](images/charles.png)
```
Comment (sotayamashita) [inline src/content/ja/fundamentals/performance/poor-connectivity/index.md]: note: use English value instead

File: src/content/ja/fundamentals/performance/http2/index.md [hunk 2]
```diff
@@ -14,254 +14,162 @@ description: HTTP/2（h2）は、ウェブでのプッシュ、多重ストリ
...
 さらに悪いことに、これによって、ヘッドオブライン ブロッキングが発生し、基盤となる TCP 接続の使用効率が低下します。
 
-
 HTTP/2 の新しいバイナリ フレーミング レイヤーではこのような制限がなくなり、クライアントとサーバーが HTTP メッセージを個々のフレームに分割し、混在させ、送信先で再構築することで、リクエストとレスポンスのフル多重化が可能です。
 
-
-
-
-![共有接続内での HTTP/2 リクエストとレスポンスの多重化](images/multiplexing01.svg)
+![HTTP/2 request and response multiplexing within a shared connection](images/multiplexing01.svg)
```
Comment (sotayamashita) [inline src/content/ja/fundamentals/performance/http2/index.md]: note: use English value instead

File: src/content/ja/fundamentals/primers/promises.md [hunk 3]
```diff
@@ -234,466 +233,461 @@ JS Promise は、最初の次の行以外すべてを無視します。
...
+  addTextToPage("Argh, broken: " + err.message);
+}
 
-    document.querySelector('.spinner').style.display = 'none'
-
-[サンプルを見る](https://googlesamples.github.io/web-fundamentals/fundamentals/getting-started/primers/sync-example.html){: target="_blank" .external }
+document.querySelector('.spinner').style.display = 'none'
+```
 
+[](https://googlesamples.github.io/web-fundamentals/fundamentals/primers/sync-example.html)[サンプルを見る](https://googlesamples.github.io/web-fundamentals/fundamentals/getting-started/primers/sync-example.html){: target="_blank" .external }
```
Comment (sotayamashita) [inline src/content/ja/fundamentals/primers/promises.md]: note: use English value instead

File: src/content/ja/tools/chrome-devtools/console/track-exceptions.md [hunk 4]
```diff
@@ -35,97 +33,67 @@ Chrome DevTools には、例外をスローしているウェブページの修
 
 どの実行パスがエラーの原因かわからない場合もあります。コンソールでは、JavaScript コールスタック全体に例外が関連付けられます。これらのコンソール メッセージを展開してスタック フレームを表示し、コード内の該当する場所に移動します。
 
-
-
-![例外のスタックトレース](images/track-exceptions-exception-stack-trace.jpg)
```
Comment (sotayamashita) [inline src/content/ja/tools/chrome-devtools/console/track-exceptions.md]: note: use English value instead

**Comment by petele** [comment]
Looks like a bunch of build failures that need to be addressed.

**Comment by sotayamashita** [comment]
@petele I fixed the test. The Purpose of pull request is to solve the conflict on GitLocalize. I just change format of markdown. I would appreciate if you could review this PR.

Thanks in advance.

File: src/content/ja/fundamentals/accessibility/semantics-aria/hiding-and-updating-content.md [hunk 5]
```diff
@@ -12,131 +11,87 @@ description: 支援技術に対してコンテンツを非表示にする
...
-まず、DOM に対して明示的に非表示の要素は、アクセシビリティ ツリーにも含まれません。したがって、CSS スタイルで `visibility:
-hidden` または `display: none` が指定されている要素、および HTML5 の `hidden` 属性が使用されている要素も、支援技術ユーザーに対して非表示になります。
-
+まず、DOM に対して明示的に非表示の要素は、アクセシビリティ ツリーにも含まれません。したがって、CSS スタイルで `visibility: hidden` または `display: none` が指定されている要素、および HTML5 の `hidden` 属性が使用されている要素も、支援技術ユーザーに対して非表示になります。
 
 ただし、視覚的にレンダリングされていなくても、明示的に非表示になっていない要素は依然としてアクセシビリティ ツリーに含まれます。よく使われるのは、絶対位置で画面外に配置された要素に「スクリーン リーダー限定テキスト」を含める方法です。
 
-
-
-    .sr-only {
```
Comment (petele) [inline src/content/ja/fundamentals/accessibility/semantics-aria/hiding-and-updating-content.md]: Why is this changing? The original EN source docs us 4 space indents

**Comment by petele** [comment]
I'm not clear what changes in here, it looks to me like it's mostly markdown styles, for example changing bullet lists from `*` to `-` and removing extra spaces. There are so many files and so many changes in here, it's hard to understand what's going on.

**Comment by petele** [comment]
I'm still not sure why we need to change the styling?

**Pull request closed without merging**